app/scripts/app-init.js

```diff
@@ -52,12 +50,13 @@ function importAllScripts() {
 
   const startImportScriptsTime = Date.now();
   // value of applyLavaMoat below is dynamically replaced at build time with actual value
-  const applyLavaMoat = true;
+  const applyLavaMoat = process.env.APPLY_LAVAMOAT;
```

Comment:
non-blocker: maybe 
```suggestion
  const applyLavaMoat = process.env.APPLY_LAVAMOAT ?? true;
```
to preserve previous default value

Reply:
Previously the `true` value was _always_ replaced by the build script, so really there was no default despite the appearances.

Fair point that `undefined` is a poor default, and would lead to confusion if we accidentally stopped injecting this. Rather than add a default, I will add a check to ensure an error is thrown if this is missing.